packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec

```diff
@@ -17,7 +17,7 @@
 
 Summary:    MaaS Discovery Plugin for Foreman
 Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    %{mainver}
+Version:    16.0.1
```

Comment:
can you please update `mainver` above instead of setting the version here?

(or remove all that mainver handling, but that's a different discussion I guess)